Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+taipy-gui = {git = "https://git@github.com/Avaiga/taipy-gui.git@develop"}
+taipy-rest = {git = "https://git@github.com/Avaiga/taipy-rest.git@develop"}
+
+[dev-packages]
+pytest = "*"
+
+[requires]
+python_version = "3.9"
```

Comment:
I though 3.8 would be the minimum...

Reply:
true, I didn't pay much attention to it because I was trying to get it up and running quickly for testing. Will switch to 3.8 :D